src/executors/default.yml

```diff
@@ -1,12 +1,4 @@
 description: >
-  This is a sample executor using Docker and Node. If you want to provide a custom environment in your orb, insert your image here.
-  If you do not require an executor, you can simply delete this directory.
+  A single complete job to run Cypress tests in your project
```

Comment:
Is the executer where we describe a job? This is just an image with cypress and node, etc right?